scripts/overviewCard.js

```diff
@@ -1,15 +1,26 @@
+import renderToDom from "./renderToDom.js";
+import fakeRepos from "./fakedata.js";
+
+
+
 //card
 const overviewDom = (array) => {
-    let domString = "";
-    for (const repo of array) {
-      domString = `
-      <div class="card">
-          <div class="card-body">
-               <h5 class="card-title">${repo.title}</h5>
-               <p class="card-text">${repo.codtype}</p>
-          </div>
-      </div>
-      `;
-    }
-    renderToDom("#overviewContainer", domString);
```

Comment:
I dont think we need a whole new container for every page, theres just the main content, (the pinned repos) and the formContent (the add a project)